src/Components/CommandPanel.js

```diff
@@ -0,0 +1,61 @@
+import { memo } from 'react';
+import PropTypes from 'prop-types';
+import { FILTER_ACTIVE, FILTER_COMPLETED, SHOW_ALL } from '../Filters';
+
+import style from './commandPanel.module.scss';
+
+function CommandPanel({ todos, onFilterChanged, onRemoveCompleted }) {
+  return (
+    <div className={style.panel}>
+      <span className={style.span_items_left}>
+        {todos !== null &&
+          todos.length > 0 &&
```

Comment:
If array is empty we will have text ` items left`?

Reply:
fixed - it will be no items left